pkg/cluster/cluster.go

```diff
@@ -301,18 +301,16 @@ func (c *Cluster) Create() error {
 func (c *Cluster) deleteMachine(machine *Machine, i int) error {
 	name := machine.ContainerName()
 
+	if machine.IsIgnite() {
+		log.Infof("Deleting machine: %s ...", name)
+		return ignite.Remove(machine.name)
+	}
+
 	if !machine.IsCreated() {
```

Comment:
The `IsCreated` check needs to come before performing the removal, otherwise Ignite throws an error about the VM not existing.